versioned_docs/version-1.3.0/install-kubeslice/kubeslice-cli/install-kubeslice.mdx

```diff
@@ -13,8 +13,27 @@ controller cluster and the worker cluster names. The file should include other r
 as described in the sample configuration file provided below. This file is essential in setting 
 up KubeSlice.
 
-Custom chart values can be specified in the topology configuration. For example, you can add `values:metrics.insecure=true` under 
+
+### Air-Gapped Installation
+
+To install KubeSlice in Air-Gapped environment use local helm charts. Upgrade to the latest 
```

Comment:
Same format here. 

To do something, <make/use/do/action verb> something... 

Add a comma after `environment`

Reply:
in an or in the air-gapped environment

Reply:
done